functions/Remove-DbaServerRoleMember.ps1

```diff
@@ -0,0 +1,194 @@
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: Role, Security, Login
+        Author: Mikey Bronowski (@MikeyBronowski), https://bronowski.it
+
+        Website: https://dbatools.io
+        Copyright: (c) 2021 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .LINK
+        https://dbatools.io/Remove-DbaServerRoleMember
+
+    .EXAMPLE
+        PS C:\> Remove-DbaServerRoleMember -SqlInstance server1 -ServerRole dbcreator -Login login1
+
+        Removes login1 from the dbcreator fixed server-level role on the instance server1.
+
+    .EXAMPLE
+        PS C:\> Remove-DbaServerRoleMember -SqlInstance server1, sql2016 -ServerRole customrole -Login login1
+
+        Removes login1 from customrole custom server-level role on the instance server1 and sql2016.
+
+    .EXAMPLE
+        PS C:\> Remove-DbaServerRoleMember -SqlInstance server1 -ServerRole customrole -Role dbcreator
+
+        Removes customrole custom server-level role from the dbcreator fixed server-level role.
+
+    .EXAMPLE
+        PS C:\> $servers = Get-Content C:\servers.txt
+        PS C:\> $servers | Remove-DbaServerRoleMember -ServerRole sysadmin -Login login1
+
+        Removes login1 from the sysadmin fixed server-level role in every server in C:\servers.txt.
+
+    .EXAMPLE
+        PS C:\> Remove-DbaServerRoleMember -SqlInstance localhost -ServerRole bulkadmin, dbcreator -Login login1
+
+        Removes login1 from the bulkadmin and dbcreator fixed server-level roles on the server localhost.
+
+    .EXAMPLE
+        PS C:\> $roles = Get-DbaServerRole -SqlInstance localhost -ServerRole bulkadmin, dbcreator
+        PS C:\> $roles | Remove-DbaServerRoleMember -Login login1
+
+        Removes login1 from the bulkadmin and dbcreator fixed server-level roles on the server localhost.
+
+    .EXAMPLE
+        PS C:\ $logins = Get-Content C:\logins.txt
+        PS C:\ $srvLogins = Get-DbaLogin -SqlInstance server1 -Login $logins
+        PS C:\ Remove-DbaServerRoleMember -Login $logins
+
+        Removes all the logins found in C:\logins.txt from mycustomrole custom server-level role on server1.
+    #>
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'High')]
+    param (
+        [parameter(ValueFromPipeline)]
+        [DbaInstance[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [parameter(ValueFromPipeline)]
+        [string[]]$ServerRole,
+        [string[]]$Login,
+        [string[]]$Role,
+        [parameter(ValueFromPipeline)]
+        [object[]]$InputObject,
+        [switch]$EnableException
+    )
+
+    begin {
+        if ( (Test-Bound SqlInstance -Not) -and (Test-Bound ServerRole -Not) -and (Test-Bound Login -Not) ) {
+            Stop-Function -Message "You must pipe in a ServerRole, Login, or specify a SqlInstance"
```

Comment:
Not sure if this message is correct.

Reply:
What's your suggestion? It's a Copy of Add-DbaServerRoleMember so might need changing there too.

Reply:
First of all: What does the if tests? It is true if there is no SqlInstance, no ServerRole and no Login. As soon as you provide two of the three, it is false and the function does not stop. Is this intended? I think it is even wrong in Add-DbaServerRoleMember. 
So I would just delete this test. And then maybe think of new tests. And then start with comments explaining what to test in plain english. And then code it.